app/controllers/provider/admin/applications_controller.rb

```diff
@@ -0,0 +1,112 @@
+# frozen_string_literal: true
+
+class Provider::Admin::ApplicationsController < FrontendController
+  include ThreeScale::Search::Helpers
+  include ApplicationsControllerMethods
+
+  before_action :authorize_partners
+  before_action :find_plans
+  before_action :find_buyer, only: :create
+  before_action :authorize_multiple_applications, only: :create
+  before_action :find_application_plan, only: :create
+  before_action :find_service, only: :create
+  before_action :find_service_plan, only: :create
+  before_action :find_cinstance, except: %i[index new create]
+
+  helper_method :accessible_services
+
+  activate_menu :audience, :applications, :listing
+
+  layout 'provider'
+
+  def index
+    super
+  end
```

Comment:
No need of this. It is redundant

Reply:
I bet code climate complained about it.

Reply:
If I define the method like this, codeclimate complaints about it.
If I remove the redundant method, then codeclimate complaints about it.

See conflicting warnings in the following image:
![Screenshot 2021-06-16 at 17 46 11 (1)](https://user-images.githubusercontent.com/11672286/123792112-8de80880-d8e0-11eb-80c7-f3e09b47ff57.png)

I personally would like to make it explicit the controller implements `index`, even if it's redundant.

Reply:
The other option is rename `index` and `create` in the module and invoke them via filters, no longer needing to call `super`.

Reply:
You can waive the issue in codeclimate if that does not make any sense ;)